PrintMessage.cpp

```diff
@@ -0,0 +1,53 @@
+#include "./PrintMessage.h"
+#include <thread>
+#include <chrono>
+#include <iostream>
+#include <map>
+
+using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+
+Language currentLanguage = ENGLISH;
+
+std::map<Language, std::map<std::string, std::string>> messages = {
+    {ENGLISH, {
+        {"HYPO_THERMIA", "Temperature is critical!"},
+        {"NEAR_HYPO", "Warning: Approaching hypothermia"},
+        {"NORMAL", "Temperature is normal."},
+        {"NEAR_HYPER", "Warning: Approaching hyperthermia"},
+        {"HYPER_THERMIA", "Temperature is critical!"},
+        {"PULSE_LOW", "Pulse Rate is too low!"},
+        {"NEAR_PULSE_LOW", "Warning: Approaching low pulse rate"},
+        {"PULSE_HIGH", "Pulse Rate is too high!"},
+        {"NEAR_PULSE_HIGH", "Warning: Approaching high pulse rate"},
+        {"SPO2_LOW", "Oxygen Saturation is too low!"},
+        {"NEAR_SPO2_LOW", "Warning: Approaching low oxygen saturation"}
+    }},
+    {GERMAN, {
```

Comment:
All these strings aren't automatically tested. The `test_cases` function does a sample of them. If those are missed, we will be alerted by a failing test - which is good.
It's hard to test all the situations in all the languages manually each time. Of course, the "understand-ability" of the translation can only be confirmed manually by native speakers.
But other things can go wrong, like missing a translation in a particular language, etc. Can you think of any asserts to safeguard against that?